packages/cli-doctor/src/healthchecks/__tests__/androidHomeEnvVariable.test.ts

```diff
@@ -1,5 +1,5 @@
 import androidHomeEnvVariables from '../androidHomeEnvVariable';
-import {NoopLoader} from '../../../../tools/loader';
+import {NoopLoader} from '@react-native-community/cli/src/tools/loader';
```

Comment:
You should not require files directly from source, but rather expose them via public API. TL;DR always require from `@react-native-community/cli`.

Reply:
I see. Makes sense